onboard/catkin_ws/src/offboard_comms/README.md

```diff
@@ -109,48 +126,6 @@ void MultiplexedBasicESC::write(int8_t speed){writeMicroseconds(map(speed, -128,
-```
-rosservice call /enable_camera false
-```
-to disable the camera. Note: this disabled state should not be maintained for more than about 3 minutes as this draws significant current from the Arduino. A warning is thrown in serial.launch indicating this. To re-enable the camera, run
-```
-rosservice call /enable_camera true
-```
-By default, the camera is enabled. The camera relay topic can be checked by running:
-```
-rostopic echo /offboard/camera_relay
-```
-This will print the current state of the topic that the Arduino is subscribed to. The camera relay status topic can be checked by running:
-```
-rostopic echo /offboard/camera_relay_status
-```
-This will print the current state of the relay itself as reported by the Arduino. The distinction between the two topics is that the camera relay topic is the state that the relay is trying to be in, while the camera relay status topic is the state that the relay is actually in. The camera relay topic is set by the service call, while the camera relay status topic is set by the Arduino. The camera relay status topic is used to check if the camera is actually enabled or disabled, while the camera relay topic is used to check if the camera is trying to be enabled or disabled. Additionally, a message is displayed in the serial.launch terminal when the camera is enabled or disabled.
-
-
-### Servos
-The servos utilize the `/offboard/set_servo_angle` service that is of type `custom_msgs/SetServo.srv`. The request to this service consists of an unsigned 8-bit integer `num` that corresponds to the pin number of the servo, and an unsigned 8 bit integer `angle` that corresponds to the desired angle (0-180). The reply consists of a bool `success` that corresponds to whether the request successfully set the angle.
-
-Values for `num` are 0-indexed (meaning the first servo corresponds to pin number 0) and values that are greater than or equal to the number of servos will result in an unsuccessfull call. Values for `angle` that are >180 will also result in an unsuccessful call.
+The pressure sensor sends raw serial data to the main computer. This data is then published to the `/sensors/depth` topic. Note that this is a separate package and will need to be run in addition to this `offboard_comms` package. The data is of type `nav_msgs/Odometry.msg`. The data is published by the `data_pub` node.
 
-This service lives in the script `servo_wrapper.py` which communicates with the Arduino by publishing a message of type `custom_msgs/ServoAngleArray.msg`. This is an array of 8 8-bit unsigned integers, and publishes them to topic `/offboard/servo_angles`.
\ No newline at end of file
+All values of the odometry message are set to 0 except for the `pose.pose.position.z` value, which is set to the depth in meters. The `pose.pose.orientation` is set to the identity quaternion. Except for the `pose.pose.position.z` value, all other values are unused in sensor fusion.
```

Comment:
No need to mention details regarding the behavior of the pressure sensor script in `data_pub`. Only mention details directly relevant to `offboard_comms`, and say that the `data_pub` package handles the rest.